src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,41 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static int driverId = 0;
+
     public static void main(String[] args) {
-        // test your code here
+        final ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        final DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+
+        manufacturerService.getAll()
+                .forEach(System.out::println);
+
+        driverService.create(getDefaultDriver());
+        driverService.create(getDefaultDriver());
+        driverService.create(getDefaultDriver());
+
+        driverService.getAll()
+                .forEach(System.out::println);
+
+        driverService.update(new Driver(3L, "defoult3", "333"));
+        driverService.update(new Driver(2L, "defoult2", "2222"));
+
+        driverService.delete(2L);
```

Comment:
```suggestion
        Driver first = driverService.create(getDefaultDriver());
        Driver secont = driverService.create(getDefaultDriver());
        Driver third = driverService.create(getDefaultDriver());

        driverService.getAll()
                .forEach(System.out::println);

        driverService.update(new Driver(third.getId(), "defoult3", "333"));
        driverService.update(new Driver(secont.getId(), "defoult2", "2222"));

        driverService.delete(secont.getId());
```

It's better not to use magic numbers